app/frontend/packs/components/checkbox_filter.js

```diff
@@ -0,0 +1,164 @@
+let CheckboxFilter = function(params) {
```

Comment:
I think we might need to think about renaming this in the real implementation to be representative of it including a search box and checkbox filters